web/app/AttributeFormatter.js

```diff
@@ -47,6 +47,10 @@ Ext.define('Traccar.AttributeFormatter', {
         return (hours + ' ' + Strings.sharedHourAbbreviation + ' ' + minutes + ' ' + Strings.sharedMinuteAbbreviation);
     },
 
+    deviceNameFormatter: function (value) {
+        return Ext.getStore('Devices').findRecord('id', value).get('name');
```

Comment:
There is a special method `getById` or something like that. Potentially it's more efficient.